docs/Bioinformatic-Analyses/RNAseq-on-Cavatica/rna_seq_7.md

```diff
@@ -28,15 +28,14 @@ Analysis with DESeq2 Public App
 * **Factor level - reference** represents the denominator for the log2 fold change (LFC) i.e what condition/group do we compare against. Enter **Ependymoma** as the reference factor. Changing the order of the reference or test factor level results in reversal of direction of log fold change.
 * **Factor level - test** represents the numerator for the LFC. Enter **Medulloblastoma** as the test factor.
 * Select the **Quantification tool** used to calculate transcript abundance from the drop down menu. The expression data for our data were generated using **kallisto**.
-* **IgnoreTxVersion** is a function in `tximport` package designed to ignore transcript version number. Set it to **True**.
-* DESeq2 allows for the shrinkage of the LFC which uses information from all genes to generate accurate estimates. Although, using LFC shrinkage does not impact the total number of genes that are identified as being significantly differentially expressed, it is useful for downstream assessment of results. Set the **log2 fold change shrinkage** to **True**.
+* DESeq2 allows for the shrinkage of the LFC which uses information from all genes to generate accurate estimates. LFC shrinkage tampers down any non-informative differences in the small count gene while the large fold changes from genes with lots of statistical information are not shrunk. This allows for comparing all estimated LFC across experiments. Set the **log2 fold change shrinkage** to **True**.
```

Comment:
hm, the LFC shrink description is still a bit confusing to me. i think it would help to focus this text on why someone would want to shrink the LFC (i.e., to reduce false positives). you could link in the DESeq2 docs for explaining how the function works..

reading the LFC shrink sections of this tutorial was helpful - 
https://hbctraining.github.io/DGE_workshop_salmon_online/lessons/04b_DGE_DESeq2_analysis.html

Reply:
The dispersion in the tutorial refers to the mean expression value and is different from the lfc shrinkage. I have modified the current text it to make it simpler.